.gitignore

```diff
@@ -0,0 +1,7 @@
+*.gcompcache
+*.lvprojectcache
+*.niaddoncache
+*.lvcodedb
+*.aliases
+*.lvlps
+*.gcompcache-journal
```

Comment:
Looks like git is already ignoring directories starting with `.` like `.cache` and the generated `PROJECTMEDIA` dir is empty and ignored by git